script.js

```diff
@@ -0,0 +1,164 @@
+const crypt={
+    secret:"CIPHERKEY",
+    encrypt : clear => {
+        const cipher = CryptoJS.AES.encrypt(clear, crypt.secret);
+        return cipher.toString();
+      },
+    decrypt : cipher => {
+        const decipher = CryptoJS.AES.decrypt(cipher, crypt.secret);
+        return decipher.toString(CryptoJS.enc.Utf8);
+      }
+};
+const cookieExist=document.cookie;
+const userCookieDeatail=cookieExist.split("=")
+if(userCookieDeatail.length===1){
```

Comment:
use truthy checking instead
```suggestion
if(userCookieDeatail.length){
```